requirements.txt

```diff
@@ -1,6 +1,6 @@
 Django >= 1.4, < 1.5
 pika
-django_nose
+django_nose=1.4.1
```

Comment:
You need a double equal sign.  This is broken.  @muhammad-ammar @symbolist  @wedaly @macdiesel FYI